### DISCUSSION TITLE
Setting A Particular Interval Range on X-Axis for DateTime series

### DISCUSSION THREAD

**Discussion by viswanath19** [apexcharts/apexcharts.js#4358] [Q&A]
Hi Team,

I just wanted to check if there is a any configurable way to add a time interval in apex chart for datetime series. 
For Eg: If I selected 24hr range and I wanted to show only labels for every 2hrs on the xaxis is there any way?

**Comment by SanAndreaP** [comment]
You can use `tickAmount` from the `xaxis` option:

```
xaxis: {
  tickAmount: 12
}
```

So in a 24hr range, if you need ticks every 2hrs => 24/2 = 12 ticks.

**Reply by JoseDelPinoIdener** [reply]
What if the axis starts at half an hour (Ex: 15:30) or you have scroll/panning enabled? How can I make sure there is a tick in each hour independent from what the Min and Max in the Xaxis are set to?
